mdpow/fep.py

```diff
@@ -1323,15 +1337,27 @@ def p_transfer(G1, G2, **kwargs):
     logger.info("[%s] transfer free energy %s --> %s calculation",
                 G1.molecule, G1.solvent_type, G2.solvent_type)
     for G in (G1, G2):
+        # for fep files generated with old code which doesn't have these attributes
+        if not hasattr(G, 'start'):
+            G.start = kwargs.pop('start', 0)
+        if not hasattr(G, 'stop'):
+            G.start = kwargs.pop('stop', None)
+        if not hasattr(G, 'SI'):
+            G.SI = kwargs.pop('SI', False)
         if kwargs['force']:
-            G.analyze(**kwargs)
+            if estimator == 'mdpow':
+                G.analyze(**kwargs)
+            elif estimator == 'alchemlyb':
+                G.analyze_alchemlyb(**kwargs)
         try:
             G.results.DeltaA.Gibbs
             G.logger_DeltaA0()
         except (KeyError, AttributeError):   # KeyError because results is a AttributeDict
             logger.warn("Must analyze simulation because no hydration free energy data available...")
-            G.analyze(**kwargs)
-
+            if estimator == 'mdpow':
+                G.analyze(**kwargs)
+            elif estimator == 'alchemlyb':
+                G.analyze_alchemlyb(**kwargs)
```

Comment:
Add `else` here ... or better: start at the beginning and check that estimator has the correct value.

Reply:
Not necessary if you check at the top, see code below.